cpp/Forces.h

```diff
@@ -122,13 +134,121 @@ inline double addAtomLJ( const Vec3d& dR, Vec3d& fout, double c6, double c12 ){
 }
 
 // Lenard-Jones force between two atoms a,b separated by vector dR = Ra - Rb
-inline double addAtomVdW( const Vec3d& dR, Vec3d& fout, double c6 ){
+inline double addAtomVdW_noDamp( const Vec3d& dR, Vec3d& fout, double c6 ){
+    double invR2 = 1/dR.norm2(); 
+    double invR4 = invR2*invR2;
+    double E     = -c6 * invR4*invR2;
+    fout.add_mul( dR , E*-6*invR2 );
+    return E;
+}
+
+// Lenard-Jones force between two atoms a,b separated by vector dR = Ra - Rb
+inline double addAtomVdW_dampConst( const Vec3d& dR, Vec3d& fout, double c6, double ADamp ){
     double r2 = dR.norm2(); r2*=r2; r2*=r2;
-    //fout.add_mul( dR , 6*c6 /( r2 + 1.0 ) );
-    //fout.add_mul( dR , 6*c6 /( r2 + 60*c6 ) );
-    fout.add_mul( dR , 6*c6 /( r2 + 180*c6 ) );
+    fout.add_mul  ( dR , 6*c6 /( r2 + ADamp*c6 ) );
+    return 0;
+}
+
+
+
+/*
```

Comment:
Do these commented-out blocks serve a purpose? I would just delete them to make it cleaner, if they are not required for later.